core/domain/suggestion_registry_test.py

```diff
@@ -3479,3 +3479,310 @@ def test_initial_object_with_valid_arguments_has_correct_properties(self):
         self.assertEqual(
             reviewable_suggestion_email_info.submission_datetime,
             self.submission_datetime)
+
+
+class TranslationReviewStatsUnitTests(test_utils.GenericTestBase):
+    """Tests for the TranslationReviewStats class."""
+
+    LANGUAGE_CODE = 'es'
+    CONTRIBUTOR_USER_ID = 'uid_01234567890123456789012345678912'
+    TOPIC_ID = 'topic_id'
+    REVIEWED_TRANSLATIONS_COUNT = 2
+    REVIEWED_TRANSLATION_WORD_COUNT = 100
+    ACCEPTED_TRANSLATIONS_COUNT = 1
+    ACCEPTED_TRANSLATIONS_WITH_REVIEWER_EDITS_COUNT = 0
+    ACCEPTED_TRANSLATIONS_WITH_REVIEWER_EDITS_WORD_COUNT = 0
+    ACCEPTED_TRANSLATION_WORD_COUNT = 50
+    FIRST_CONTRIBUTION_DATE = datetime.date.fromtimestamp(1616173836)
+    LAST_CONTRIBUTION_DATE = datetime.date.fromtimestamp(1616173836)
+
+    def test_create_translation_review_stats(self):
+        expected_stats_dict = {
+            'language_code': self.LANGUAGE_CODE,
+            'contributor_user_id': self.CONTRIBUTOR_USER_ID,
+            'topic_id': self.TOPIC_ID,
+            'reviewed_translations_count': self.REVIEWED_TRANSLATIONS_COUNT,
+            'reviewed_translation_word_count': (
+                self.REVIEWED_TRANSLATION_WORD_COUNT),
+            'accepted_translations_count': self.ACCEPTED_TRANSLATIONS_COUNT,
+            'accepted_translations_with_reviewer_edits_count': (
+                self.ACCEPTED_TRANSLATIONS_WITH_REVIEWER_EDITS_COUNT),
+            'accepted_translations_with_reviewer_edits_word_count': (
+                self.ACCEPTED_TRANSLATIONS_WITH_REVIEWER_EDITS_WORD_COUNT),
+            'first_contribution_date': self.FIRST_CONTRIBUTION_DATE,
+            'last_contribution_date': self.LAST_CONTRIBUTION_DATE,
+        }
+
+        observed_stats = suggestion_registry.TranslationReviewStats(
```

Comment:
```suggestion
        actual_stats = suggestion_registry.TranslationReviewStats(
```

Nit: Usually use "expected" and "actual" in tests. Ditto below.

Reply:
Done